lib/common/utils.c

```diff
@@ -162,6 +162,15 @@ check_number(const char *value)
 }
 
 gboolean
+check_positive_number(const char* value)
+{
+    if (check_number(value) && (value[0] != '-') && !(safe_str_eq(value,"0"))) {
```

Comment:
This is a good test, but to be extra safe, I'm thinking it should be:

    if (safe_str_eq(value, INFINITY_S) || (crm_int_helper(value, NULL) > 0)) {

The difference is that crm_int_helper() allows whitespace before the number (because strtol() does). It's probably not an issue in practice, but it doesn't hurt to be on the safe side.